extensions/ql-vscode/src/variant-analysis/variant-analysis-manager.ts

```diff
@@ -229,54 +230,12 @@ export class VariantAnalysisManager
         return;
       }
 
-      progress({
-        maxStep: 8,
-        step: 1,
-        message: "Downloading query pack",
-      });
-
-      const packName = `codeql/${language}-queries`;
-      const packDownloadResult = await this.cliServer.packDownload([packName]);
-      const downloadedPack = packDownloadResult.packs[0];
-
-      const packDir = join(
-        packDownloadResult.packDir,
-        downloadedPack.name,
-        downloadedPack.version,
-      );
-
-      progress({
-        maxStep: 8,
-        step: 2,
-        message: "Resolving queries in pack",
-      });
-
-      const suitePath = join(
-        packDir,
-        "codeql-suites",
-        `${language}-code-scanning.qls`,
-      );
-      const resolvedQueries = await this.cliServer.resolveQueries(suitePath);
-
-      const problemQueries =
-        await this.filterToOnlyProblemQueries(resolvedQueries);
-
-      if (problemQueries.length === 0) {
-        void this.app.logger.showErrorMessage(
-          `Unable to trigger variant analysis. No problem queries found in published query pack: ${packName}.`,
-        );
-        return;
-      }
-
-      const qlPackFilePath = await getQlPackFilePath(packDir);
-
       // Build up details to pass to the functions that run the variant analysis.
-      const qlPackDetails: QlPackDetails = {
-        queryFiles: problemQueries,
-        qlPackRootPath: packDir,
-        qlPackFilePath,
+      const qlPackDetails = await getCodeScanningPack(
```

Comment:
Shouldn't we still call `progress` before executing this? Something like:

```suggestion
      progress({
        maxStep: 8,
        step: 1,
        message: "Downloading and resolving queries",
      });

      const qlPackDetails = await getCodeScanningPack(
```